en/assembly/mount_gps_compass.md

```diff
@@ -21,3 +21,7 @@ You must mount the compass in a supported orientation!
 If you mount the compass at an orientation that isn't supported, for example `Yaw 30`, PX4 will detect the closest supported value.
 This will result in errors/warnings, even if the calibration appeared to succeed.
 :::
+
+## Position
+
+In order to compensate for the relative motion between the receiver and the CoG, [configure](../advanced_config/parameters.md) the following parameters to set the offsets: [EKF2_GPS_POS_X](../advanced_config/parameter_reference.md#EKF2_GPS_POS_X), [EKF2_GPS_POS_Y](../advanced_config/parameter_reference.md#EKF2_GPS_POS_Y) and [EKF2_GPS_POS_Z](../advanced_config/parameter_reference.md#EKF2_GPS_POS_Z).
```

Comment:
@bresch same questions for GPS - What would you "see" given an offset - say of 10cm or 2m - i.e how big a difference does it make? What are the conditions in which it would make a bigger difference? 

My guess is that this means you're true position will be off by the amount in the offset. Whether that matters depends on how accurate you expect that position to be - I imagine it would normally not matter too much?

Reply:
That's awesome - thanks!